src/Event/ExtensionsListener.php

```diff
@@ -47,7 +48,7 @@ public function implementedEvents(): array
     /**
      * Event handler.
      *
-     * @param ConstructEvent $event Event.
+     * @param \WyriHaximus\TwigView\Event\ConstructEvent $event Event.
```

Comment:
Why are you using a full class here instead of using the imported at the top of the file?

Reply:
Cake's convention is to use FQCN in docblocks, it makes for non ambiguous API docs.

Reply:
Check :+1: